common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -655,6 +662,10 @@ div.problem {
       font-style: italic;
       -webkit-font-smoothing: antialiased;
     }
+
+    @media print {
```

Comment:
Will this hide how many checks are remaining, or other information that appear after the button? It's reasonable to hide the buttons but other information should be preserved.